modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,241 @@
+* `Microsoft.Network/networkSecurityGroups/securityRules/write`
+* `Microsoft.Network/networkSecurityGroups/write`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/A/write`
+* `Microsoft.Network/privateDnsZones/A/delete`
+* `Microsoft.Network/privateDnsZones/SOA/read`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/write`
+* `Microsoft.Network/privateDnsZones/write`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/publicIPAddresses/join/action`
+* `Microsoft.Network/publicIPAddresses/read`
+* `Microsoft.Network/publicIPAddresses/write`
+* `Microsoft.Network/virtualNetworks/join/action`
+* `Microsoft.Network/virtualNetworks/read`
+* `Microsoft.Network/virtualNetworks/subnets/join/action`
+* `Microsoft.Network/virtualNetworks/subnets/read`
+* `Microsoft.Network/virtualNetworks/subnets/write`
+* `Microsoft.Network/virtualNetworks/write`
+====
+
+.Required permissions for checking health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/InProgress/action`
+* `Microsoft.Resourcehealth/healthevent/Pending/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for creating resource group
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourceGroups/read`
+* `Microsoft.Resources/subscriptions/resourcegroups/write`
+====
+
+.Required permissions for creating resource tags
+[%collapsible]
+====
+* `Microsoft.Resources/tags/write`
+====
+
+.Required permissions for creating storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/blobServices/read`
+* `Microsoft.Storage/storageAccounts/blobServices/containers/write`
+* `Microsoft.Storage/storageAccounts/fileServices/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/read`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/write`
+* `Microsoft.Storage/storageAccounts/fileServices/shares/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+* `Microsoft.Storage/storageAccounts/read`
+* `Microsoft.Storage/storageAccounts/write`
+====
+
+.Required permissions for creating deployment
+[%collapsible]
+====
+* `Microsoft.Resources/deployments/read`
+* `Microsoft.Resources/deployments/write`
+* `Microsoft.Resources/deployments/validate/action`
+* `Microsoft.Resources/deployments/operationstatuses/read`
+====
+
+.Optional permissions for creating marketplace virtual machine resources
+[%collapsible]
+====
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/read`
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/write`
+====
+
+.Optional permissions for enabling user-defined disk encryption set
```

Comment:
I searched in UPI doc, it doesn't mention to enable disk encryption, so I think we can remove this part from UPI doc. When doc describes this part here, then add them back.

@chiragkyal wdyt?

Reply:
Double checked with @sunzhaohua2  from cloud team, the day-2 operation on machine-api (including disk encryption set) is same on both ipi and upi, no difference. @sunzhaohua2 please correct me if I 'm wrong.

So I'm okay to keep this part as day2 operation.

@xenolinux if no more comments here, same update should be here as IPI to reword `Optional permissions for enabling user-defined disk encryption set`.

Reply:
Let's be consistent throughout the docs. We haven't called day 1 and day 2 terms. So, I think `Optional permissions for enabling user-defined disk encryption set` would make much more sense.